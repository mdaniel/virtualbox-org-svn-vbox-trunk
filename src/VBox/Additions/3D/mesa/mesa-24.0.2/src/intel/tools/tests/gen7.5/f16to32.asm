f16to32(8)      g124<1>F        g2<16,8,2>UW                    { align1 1Q };
f16to32(16)     g120<1>F        g10<16,8,2>UW                   { align1 1H };
