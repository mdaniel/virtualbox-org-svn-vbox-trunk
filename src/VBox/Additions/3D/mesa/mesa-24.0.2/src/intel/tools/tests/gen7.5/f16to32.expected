14 00 60 00 3d 01 80 2f 40 00 ae 00 00 00 00 00
14 00 80 00 3d 01 00 2f 40 01 ae 00 00 00 00 00
